arch/unitroot/unitroot.py

```diff
@@ -1714,3 +1714,63 @@ def kpss_crit(stat, trend='c'):
     crit_value = interp(cv, y[::-1], x[::-1])
 
     return pvalue, crit_value
+
+  def BandWidthNW(y, kernel = "ba"):
+    '''
+    | Automatic bandwidth selection of Andrews (1991) and of Newey and West (1994)
```

Comment:
No `|` in doc string.  The docstring must follow the NumPy style

https://numpydoc.readthedocs.io/en/latest/format.html

with sections 

```
Parameters
----------

Returns
-------

Notes
-----
```
etc.